derp/derp_server.go

```diff
@@ -1398,3 +1419,26 @@ func writePublicKey(bw *bufio.Writer, key *key.Public) error {
 	}
 	return nil
 }
+
+type ipBandwidth struct {
+	ip        string
```

Comment:
netaddr.IP ?

Reply:
I can make this into an actual IP, but it seems a bit odd that it's not an IP on the `sclient`